src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests3.cs

```diff
@@ -3052,60 +3053,493 @@ private static void Test(decimal percent)
+        >= 2 and < 4 =>  LifeStage.Toddler,
+        >= 65 =>    LifeStage.LateAdult,
+        >= 40 and < 65 => LifeStage.MiddleAdult,
+    };
+}
+enum LifeStage
+{
+    Prenatal,
+    Infant,
+    Toddler,
+    EarlyChild,
+    MiddleChild,
+    Adolescent,
+    EarlyAdult,
+    MiddleAdult,
+    LateAdult,
+}
+";
+            var expectedOutput =
+@"-1 -> Prenatal
+0 -> Infant
+1 -> Infant
+2 -> Toddler
+3 -> Toddler
+4 -> EarlyChild
+5 -> EarlyChild
+6 -> MiddleChild
+7 -> MiddleChild
+11 -> MiddleChild
+12 -> Adolescent
+13 -> Adolescent
+19 -> Adolescent
+20 -> EarlyAdult
+21 -> EarlyAdult
+39 -> EarlyAdult
+40 -> MiddleAdult
+41 -> MiddleAdult
+64 -> MiddleAdult
+65 -> LateAdult
+66 -> LateAdult
+68 -> LateAdult
+70 -> LateAdult
+80 -> LateAdult
+";
+            var compilation = CreateCompilation(source, options: TestOptions.DebugExe, parseOptions: TestOptions.Regular.WithLanguageVersion(LanguageVersion.Preview));
+            compilation.VerifyDiagnostics(
+                );
+            var compVerifier = CompileAndVerify(compilation, expectedOutput: expectedOutput);
+            compVerifier.VerifyIL("C.LifeStageAtAge", @"
+    {
+      // Code size       80 (0x50)
+      .maxstack  2
+      .locals init (LifeStage V_0)
+      IL_0000:  ldarg.0
+      IL_0001:  ldc.i4.s   20
+      IL_0003:  blt.s      IL_0011
+      IL_0005:  ldarg.0
+      IL_0006:  ldc.i4.s   40
+      IL_0008:  blt.s      IL_002a
+      IL_000a:  ldarg.0
+      IL_000b:  ldc.i4.s   65
+      IL_000d:  bge.s      IL_0046
+      IL_000f:  br.s       IL_004a
+      IL_0011:  ldarg.0
+      IL_0012:  ldc.i4.4
+      IL_0013:  blt.s      IL_0020
+      IL_0015:  ldarg.0
+      IL_0016:  ldc.i4.s   12
+      IL_0018:  bge.s      IL_002e
+      IL_001a:  ldarg.0
+      IL_001b:  ldc.i4.6
+      IL_001c:  blt.s      IL_0032
+      IL_001e:  br.s       IL_0036
+      IL_0020:  ldarg.0
+      IL_0021:  ldc.i4.0
+      IL_0022:  blt.s      IL_003a
+      IL_0024:  ldarg.0
+      IL_0025:  ldc.i4.2
+      IL_0026:  blt.s      IL_003e
+      IL_0028:  br.s       IL_0042
+      IL_002a:  ldc.i4.6
+      IL_002b:  stloc.0
+      IL_002c:  br.s       IL_004e
+      IL_002e:  ldc.i4.5
+      IL_002f:  stloc.0
+      IL_0030:  br.s       IL_004e
+      IL_0032:  ldc.i4.3
+      IL_0033:  stloc.0
+      IL_0034:  br.s       IL_004e
+      IL_0036:  ldc.i4.4
+      IL_0037:  stloc.0
+      IL_0038:  br.s       IL_004e
+      IL_003a:  ldc.i4.0
+      IL_003b:  stloc.0
+      IL_003c:  br.s       IL_004e
+      IL_003e:  ldc.i4.1
+      IL_003f:  stloc.0
+      IL_0040:  br.s       IL_004e
+      IL_0042:  ldc.i4.2
+      IL_0043:  stloc.0
+      IL_0044:  br.s       IL_004e
+      IL_0046:  ldc.i4.8
+      IL_0047:  stloc.0
+      IL_0048:  br.s       IL_004e
+      IL_004a:  ldc.i4.7
+      IL_004b:  stloc.0
+      IL_004c:  br.s       IL_004e
+      IL_004e:  ldloc.0
+      IL_004f:  ret
+    }
+");
+        }
+
+        [Fact]
+        public void RelationalFuzz_2020202_2_F()
```

Comment:
>Fuzz_2020202_2_F [](start = 30, length = 16)

Consider renaming since fuzzing is not involved.

Reply:
This was generated by the fuzz test with the parameters shown in the name.

---
In reply to: [392365148](https://github.com/dotnet/roslyn/pull/42313#discussion_r392365148) [](ancestors = 392365148)

Reply:
This may be worth a comment in case we need to maintain this test